CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+This is a simple guide of the workflow being used in Cryostat. We will use https://github.com/cryostatio/cryostatio.github.io as an example.
```

Comment:
Not sure if it should be done here or in another file/pr, but I would mention how to structure the pull request titles, and labels, and such.

Reply:
Not sure but this documentation in this file seems to explains pretty well: https://www.conventionalcommits.org/en/v1.0.0/

Reply:
As for label, do you think they are self-explained enough tho :d?

Reply:
Sure I guess that part makes sense not to include.